include/internal/quic_channel.h

```diff
@@ -448,6 +448,10 @@ uint64_t ossl_quic_channel_get_max_idle_timeout_peer_request(const QUIC_CHANNEL
 /* Get the idle timeout actually negotiated. */
 uint64_t ossl_quic_channel_get_max_idle_timeout_actual(const QUIC_CHANNEL *ch);
 
+int ossl_quic_bind_channel(QUIC_CHANNEL *ch, const BIO_ADDR *peer,
```

Comment:
I think function deserves a comment. This is my suggestion:
```
+/*
+ * Bind channel instance connection ids, used when server validates client
+ * with retry packet. In this case we know connection ids (CIDs) before
+ * channel is created.
+ */
```

Reply:
added a comment to the funciton definition, which is where I think all internal functions need to be documented per coding style guide